src/Bridge/Doctrine/Orm/Filter/OrderFilter.php

```diff
@@ -143,6 +143,11 @@ protected function filterProperty(string $property, $direction, QueryBuilder $qu
      */
     protected function extractProperties(Request $request/*, string $resourceClass*/): array
     {
-        return $request->query->get($this->orderParameterName, []);
+        $properties = $request->query->get($this->orderParameterName, []);
+        if (!is_array($properties)) {
+            return [];
```

Comment:
Hmm this means that the parameter will not be correctly parsed. Shouldn't this be seen as an error? I mean, if nothing shows up, the end user will not know that this isn't giving the expecting results.